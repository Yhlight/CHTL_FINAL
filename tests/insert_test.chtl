[Template] @Element MyContent {
    h2 { text { "Original Title" } }
    p { text { "Original paragraph." } }
}

html {
    body {
        @Element MyContent {
            insert before h2[0] {
                h1 { text { "New Main Title" } }
            }

            insert after p[0] {
                hr;
                p { text { "An inserted paragraph." } }
            }
        }
    }
}
