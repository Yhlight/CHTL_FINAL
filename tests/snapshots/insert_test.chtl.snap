sh: 1: chtl_compiler: not found
